generators/client/templates/angular/src/main/webapp/app/shared/language/_language.helper.ts

```diff
@@ -32,12 +20,13 @@ export class JhiLanguageHelper {
      * 3. 'global.title'
      */
     updateTitle(titleKey?: string) {
-        // TODO : Proper way to handle this.
-        // if (!titleKey && this.$state.current.data && this.$state.current.data.pageTitle) {
-        //     titleKey = this.$state.current.data.pageTitle;
-        // }
-        // this.translateService.get(titleKey || 'global.title').subscribe(title => {
-        //     window.document.title = title;
-        // });
+        
```

Comment:
It's no longer needed to use `window` to set title, we can use the `TitleService`

https://angular.io/docs/ts/latest/cookbook/set-document-title.html